src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![feature(plugin_registrar, quote, rustc_private, core)]
+#![feature(plugin_registrar, quote, rustc_private, convert)]
```

Comment:
Using deref coercions might also let us drop this feature gate, not that it matters since this is a compiler plugin and therefore epically unstable at all times.